packages/gatsby-theme-docs/src/layouts/internals/layout-release-note.js

```diff
@@ -1,27 +1,78 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { MDXRenderer } from 'gatsby-plugin-mdx';
+import styled from '@emotion/styled';
 import SpacingsStack from '@commercetools-uikit/spacings-stack';
-import { Markdown } from '@commercetools-docs/ui-kit';
+import { Markdown, designSystem } from '@commercetools-docs/ui-kit';
 
 const ReleaseNoteTitle = Markdown.withAnchorLink(Markdown.H3);
+const DateElement = styled.div`
+  line-height: 16px;
+`;
+const Type = styled.p`
+  display: inline-block;
+  padding: ${designSystem.dimensions.spacings.xs}
+    ${designSystem.dimensions.spacings.s};
+  font-size: 12px;
+  line-height: 16px;
+  border: 1px solid #00ccb4;
+  border-radius: 2px;
+  background-color: #b3fff6;
```

Comment:
No magic numbers please. All those values should be defined in the design system, and I presume most of them already are.